models/article-models.js

```diff
@@ -34,7 +34,13 @@ exports.returnArticles = (query) => {
     return Promise.reject({ status: 400, msg: "Bad request" });
   }
 
-  let queryStr = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id`;
+  let queryStr = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id`;
+
+  if (topic) {
+    queryStr += ` WHERE articles.topic = '${topic}' GROUP BY articles.article_id`;
```

Comment:
This is currently open to SQL injection, so please add some code to prevent this.